### DISCUSSION TITLE
Successful build, tests successful, but njoy21 "command not found"

### DISCUSSION THREAD

**Discussion by ajbiffl219** [njoy/NJOY21#160] [Q&A] [answered]
I've just followed the setup and building instructions on https://docs.njoy21.io/install.html to get NJOY2021 up and running

The tests all complete successfully, but my system (WSL on Ubuntu 20.04) doesn't even recognize the command "njoy21"

I feel like I must be missing something fundamental. What am I doing wrong?

More info on my system:
GNU 9.4.0
python 3.8.5
git 2.25.1

**Comment by whaeck** [comment, marked as answer]
Hi,


With the current build system, the system's path is not changed for that to work directly. You will need to modify/create a .bashrc or .bash_profile file (or the equivalent for the shell you are using) with the following line in it:


export PATH=/path/to/njoy/bin:$PATH


where '/path/to/njoy/bin' is obviously the path on your WSL mount where the executable is.


Best regards,

Wim

________________________________
From: AJ Biffl ***@***.***>
Sent: Sunday, March 20, 2022 2:22:20 PM
To: njoy/NJOY21
Cc: Subscribed
Subject: [EXTERNAL] [njoy/NJOY21] Successful build, tests successful, but njoy21 "command not found" (Discussion #160)


I've just followed the setup and building instructions on https://docs.njoy21.io/install.html to get NJOY2021 up and running

The tests all complete successfully, but my system (WSL on Ubuntu 20.04) doesn't even recognize the command "njoy21"

I feel like I must be missing something fundamental. What am I doing wrong?

More info on my system:
GNU 9.4.0
python 3.8.5
git 2.25.1

—
Reply to this email directly, view it on GitHub<https://github.com/njoy/NJOY21/discussions/160>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AENOHQAU4EQZAXEJNQKJR63VA6CHZANCNFSM5RF7JINQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
